src/cryptoadvance/specter/hwi_rpc.py

```diff
@@ -417,6 +418,9 @@ def _get_client(
                 if chain == "liquidv1":
                     chain = "main"
                 client.chain = Chain.argparse(chain)
+                if type(client) is JadeClient:
+                    if chain == "signet":
+                        client.chain = Chain.TEST
```

Comment:
If Jade requires a replacement of signet to testnet, does it also apply to regtest or it works fine with regtest?

Reply:
No regtest works fine